src/app/routes/cpsAsset/utils/getAdditionalPageData.js

```diff
@@ -17,6 +20,61 @@ import fetchPageData from '../../utils/fetchPageData';
 
 const noop = () => {};
 
+// 004_brasil_recommendations_experiment
+const getRecommendations = (service, variant, assetUri) => {
+  if (service !== 'portuguese') {
+    return [
+      {
+        name: 'recommendations',
+        path: getRecommendationsUrl({ assetUri, variant }),
+        assetUri,
+        api: 'recommendations',
+        apiContext: 'secondary_data',
+      },
+    ];
+  }
+
+  return [
+    {
+      name: 'recommendations',
+      attachAgent: true,
```

Comment:
You could probably simplify this logic to always attach the agent and always fetch recommendations from the internal endpoint. This is actually something we were looking to do anyway.

Reply:
I agree, I'll create a separate ticket for that. That's probably outside the scope of this ticket.